src/modules/pua_dialoginfo/pua_dialoginfo.c

```diff
@@ -856,6 +844,9 @@ static int mod_init(void)
 		LM_DBG("configured to use headers for uri values\n");
 	}
 
+	if(publish_dialog_req_within)
+		dialog_event_types |= DLGCB_REQ_WITHIN;
```

Comment:
This will change the callback types, it will be not synchronized with the structure inside the dialog module.